admin/includes/cmb2-extra.php

```diff
@@ -24,6 +24,12 @@ function ds_cmb_render_multicheck_posttype( $field, $escaped_value, $object_id,
 			if ( in_array( $cpt, $values ) ) {
 				$args[ 'checked' ] = 'checked';
 			}
+
+			// @TODO Possibly find a solution if CMB2 trunk commit 1b3e251f8db429b3930e44ff43495838a9a39744
```

Comment:
ok so the problem is cmb 2.2.* branch.

Try:
`CMB2_Type_Multi_Base::list_input( $args, $i );`